test-projects/typescript-i/configs/eslint.config.js

```diff
@@ -4,5 +4,5 @@ const sheriff = require('@softarc/eslint-plugin-sheriff');
 
 module.exports = tseslint.config({
   files: ['**/*.ts'],
-  extends: [sheriff.configs.all],
+  extends: [...sheriff.configs.all],
```

Comment:
Same as https://github.com/softarc-consulting/sheriff/pull/179/files#r2040325564

Reply:
Reverted. No longer a question. I've chosen to use file exclusion in the code rather than inside eslint config.